suite/interfaces.go

```diff
@@ -1,6 +1,8 @@
 package suite
 
-import "testing"
+import (
```

Comment:
To maintain consistency, wouldn't it be better to remove the parenthesis when we're importing just one package?

Reply:
I think `goimports` added it automatically here, sorry! Fixing.